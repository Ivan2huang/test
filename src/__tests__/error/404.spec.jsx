import React from 'react';

import { render } from '@testing-library/react';

import NotFoundPage from '../../pages/error/404';

jest.mock('../../helpers/helpers', () => ({
  formatMessage: jest.fn((intl, id, defaultMessage) => defaultMessage),
}));

jest.mock('../../modules/error/Error', () => () => (
  <div>not found component</div>
));

describe('Not Found Page', () => {
  it('should match snapshot', () => {
    const { container } = render(<NotFoundPage intl={{}} />);
    expect(container).toMatchSnapshot();
  });
});
