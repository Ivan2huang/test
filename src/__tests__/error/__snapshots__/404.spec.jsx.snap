// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page should match snapshot 1`] = `
<div>
  <div>
    not found component
  </div>
</div>
`;
